SECURITY.md

```diff
@@ -0,0 +1,174 @@
+- [Security Posture](#security-posture)
+- [Security Team](#security-team)
+- [Security Policy Updates](#security-policy-updates)
+
+<!-- tocstop -->
+
+## Supported Versions
+
+The following versions are currently supported and receive security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 1.2.x   | :white_check_mark: |
+| 1.1.x   | :x:                |
+| 0.0.x   | :x:                |
+
+## Reporting a Vulnerability
+
+We're extremely grateful for security researchers and users that report
+vulnerabilities to us. All reports are thoroughly investigated by the project
+[security team](#security-team).
+
+Vulnerabilities are reported privately via GitHub's
+[Security Advisories](https://docs.github.com/en/code-security/security-advisories)
+feature. Please use the following link to submit your vulnerability:
+[Report a vulnerability](https://github.com/slsa-framework/slsa-github-generator/security/advisories/new)
+
+Please see
+[Privately reporting a security vulnerability](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability#privately-reporting-a-security-vulnerability)
+for more information on how to submit a vulnerability using GitHub's interface.
+
+### When Should I Report a Vulnerability?
+
+- You think you discovered a potential security vulnerability in slsa-github-generator
+- You are unsure how a vulnerability affects slsa-github-generator
+- You think you discovered a vulnerability in another project that slsa-github-generator depends on
+  - For projects with their own vulnerability reporting and disclosure process, please report it directly there
+
+### When Should I NOT Report a Vulnerability?
+
+- You need help tuning GitHub Actions for security
+- You need help applying security related updates
+- Your issue is not security related
+
+### Vulnerability Response
+
+Each report is acknowledged and analyzed by the [Security Team](#security-team)
+within 14 days. This will set off the
+[Security Release Process](#security-release--disclosure-process).
+
+Any vulnerability information shared with the Security Team stays within
+slsa-github-generator project and will not be disseminated to other projects
+unless it is necessary to get the issue fixed.
+
+As the security issue moves from triage, to identified fix, to release planning
+we will keep the reporter updated.
+
+## Security Release & Disclosure Process
+
+Security vulnerabilities should be handled quickly and sometimes privately. The
+primary goal of this process is to reduce the total time users are vulnerable
+to publicly known exploits.
+
+### Private Disclosure
+
+We ask that all suspected vulnerabilities be privately and responsibly
+disclosed via the [private disclosure process](#reporting-a-vulnerability)
+outlined above.
+
+Fixes may be developed and tested by the [Security Team](#security-team) in
+private repositories if deemed necessary.
```

Comment:
Do we mean private from the reporter? Or the GitHub Security Advisory private branches which are private from the general public? I think the latter should be the default way we handle fixes for a reported vulnerability.

Reply:
Yes, I meant the latter. I'll clarify.

Reply:
Done.